catalyst/metrics/tests/test_cmc_metric.py

```diff
@@ -65,6 +69,68 @@
     for i in range(1, 101, 10)
 )
 
+TEST_DATA_SIMPLE_MASKED = (
```

Comment:
it is really hard to understand "who is who". could you please use dict with keys, add comments or use named tuple? we definitely need names for this piece of code)